tests/js/spec/views/issueList/overview.spec.jsx

```diff
@@ -674,358 +603,652 @@ describe('IssueList', function () {
           query: {
-            project: [],
             environment: [],
+            project: ['3559'],
             statsPeriod: '14d',
             sort: 'date',
           },
         })
       );
 
-      wrapper.setProps({
-        params: {
-          ...router.params,
-          searchId: '234',
-        },
-      });
+      expect(wrapper.getByTestId('saved-search-title')).toHaveTextContent(
+        'Unresolved TypeErrors'
+      );
 
-      expect(getSavedSearchTitle(wrapper)).toBe('Assigned to Me');
+      fireEvent.click(wrapper.getByLabelText('Pin this search'));
 
-      createPin = MockApiClient.addMockResponse({
-        url: '/organizations/org-slug/pinned-searches/',
-        method: 'PUT',
-        body: {
-          ...assignedToMe,
-          isPinned: true,
-        },
+      await waitFor(() => {
+        expect(createPin).toHaveBeenCalled();
       });
 
-      wrapper.find('Button[aria-label="Pin this search"] button').simulate('click');
+      fireEvent.click(wrapper.getByLabelText('Unpin this search'));
 
-      expect(createPin).toHaveBeenCalled();
+      await waitFor(() => {
+        expect(deletePin).toHaveBeenCalled();
+      });
+    });
 
-      await tick();
-      wrapper.update();
+    it('does not allow pagination to "previous" while on first page', async function () {
+      createWrapper();
 
-      expect(browserHistory.push).toHaveBeenLastCalledWith(
-        expect.objectContaining({
-          pathname: '/organizations/org-slug/issues/searches/234/',
-        })
+      await waitFor(() => {
+        expect(wrapper.queryByTestId('loading-indicator')).toBe(null);
+      });
+
+      expect(wrapper.getByLabelText('Previous').getAttribute('aria-disabled')).toEqual(
+        'true'
       );
 
-      wrapper.setProps({
-        params: {
-          ...router.params,
-          searchId: '234',
+      issuesRequest = MockApiClient.addMockResponse({
+        url: '/organizations/org-slug/issues/',
+        body: [group],
+        headers: {
+          Link: '<http://127.0.0.1:8000/api/0/organizations/org-slug/issues/?cursor=1443575000:0:0>; rel="previous"; results="true"; cursor="1443575000:0:1", <http://127.0.0.1:8000/api/0/organizations/org-slug/issues/?cursor=1443574000:0:0>; rel="next"; results="true"; cursor="1443574000:0:0"',
         },
       });
 
-      await tick();
-      wrapper.update();
+      fireEvent.click(wrapper.getByLabelText('Next'));
 
-      expect(getSavedSearchTitle(wrapper)).toBe('Assigned to Me');
+      expect(browserHistory.push).toHaveBeenLastCalledWith({
+        pathname: '/organizations/org-slug/issues/',
+        query: {
+          cursor: '1443575000:0:0',
+          page: 1,
+          environment: [],
+          project: [],
+          query: 'is:unresolved',
+          statsPeriod: '14d',
+        },
+      });
     });
 
-    it('pinning and unpinning searches should keep project selected', async function () {
```

Comment:
Was this test removed?

Reply:
i've combined this one with `pins and unpins a saved query` it just checks the project id now too